content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
```

Comment:
- When I go to https://gitlab.com/iterative.ai/example_cml I don't see any "Fork" link or button.
- Minor: probably no need to use word "fork" 3 times.

Reply:
> I don't see any "Fork" link

Oh, I wasn't signed in. Maybe mention that? Minor

Reply:
- It's in the upper right hand side.  There's a pic the with bounding box to clarify.
- LOL.  True. Let me see if I can figure out how you did that change in here....